src/routes/jira/jira-router.ts

```diff
@@ -15,3 +17,5 @@ JiraRouter.post("/sync", JiraContextJwtTokenMiddleware, JiraSyncPost);
 JiraRouter.use("/events", JiraEventsRouter);
 JiraRouter.use("/select-version", JiraSelectVersionRouter);
 JiraRouter.use("/server-url", JiraServerUrlRouter);
+JiraRouter.use("/app-creation", JiraAppCreationRouter);
+JiraRouter.use("/verify-server-url", JiraVerifyServerUrlRouter);
```

Comment:
But both JiraAppCreationRouter and JiraVerifyServerUrlRouter call .post. We can't have 2 posts in one router can we? An alternative is to put the verification step and the db query check in the same file instead of splitting it out. Thoughts?